docs/basics/options.md

```diff
@@ -1083,3 +1088,52 @@ searchLimit: undefined, // disable search count limits
 
 searchLimit: 150,
 ```
+
+### `legacyLinking`
+
+| Config file name | CLI short form | CLI long form      | Format    | Default |
+| ---------------- | -------------- | ------------------ | --------- | ------- |
+| `legacyLinking`  |                | `--legacy-linking` | `boolean` | `false` |
+
+Set this to `true` to use the flat-folder style linking previously used in v5. This option
+will otherwise link any matches to a tracker specific folder inside of `linkDir` (if set).
+
+:::warning qBittorrent
+Users of qBittorrent with AutoTorrentManagement enabled will need to set this to `true`.
+:::
+
+#### `legacyLinking` Examples (CLI)
+
+```shell
+cross-seed search --legacy-linking
+```
+
+#### `legacyLinking` Examples (Config file)
+
+```js
+legacyLinking: true,
+
+legacyLinking: false,
+```
+
+### `blockList`
+
+| Config file name | CLI short form           | CLI long form            | Format      | Default |
+| ---------------- | ------------------------ | ------------------------ | ----------- | ------- |
+| `blockList`      | `--block-list <strings>` | `--block-list <strings>` | `string(s)` |         |
+
+`cross-seed` will exclude any of the files/releases from cross-seeding during the prefiltering done
+at startup. You can include keywords, infoHashes, or complete torrent/file names.
+
+#### `blockList` Examples (CLI)
+
+```shell
+cross-seed search --block-list "badRelease","blockedGroup","595ceca24d075435435313c319c3a5f3bec3965a"
```

Comment:
array options on the command line are separated by spaces

```suggestion
cross-seed search --block-list badRelease blockedGroup 595ceca24d075435435313c319c3a5f3bec3965a
```

Reply:
Copy-paste laziness on my part.